trajectory_generator:
  rover: 'RR01'
  dt: .02
  num_timesteps: 100
  goal_states: [-1.0, 0.0, 0.0, 0.0]
  x_bounds: [[-10.0, 10], [-10.0, 10.0], [-1.0, 1.0], [-inf, inf]]
  u_bounds: [[-1.0, 1.0], [-0.785, 0.785]]
  terminal_cost_weight: 1.0
  waypoints_cost_weight: 1.0
  input_cost_weight: 1.0
  travel_cost_weight: 1.0
  travel_dist_cost_weight: 0.001
  input_rate_cost_weight: 1.0 
  collision_cost_weight: 1.0